tests/cypress/utils/graphqlUtils.ts

```diff
@@ -0,0 +1,101 @@
+export class GraphqlUtils {
```

Comment:
Some function deleteNode or getNode are already implemented in @jahia-cypress, have a look to this file https://github.com/Jahia/jahia-cypress/blob/main/src/utils/Utils.ts
Don't hesitate to update jahia cypress to add generic functions that could be used by several projects